client-v2/src/components/FrontsCAPIInterface/Feed.tsx

```diff
@@ -28,31 +29,24 @@ const Feed = ({ articles = [], error }: FeedProps) => (
     {articles.length ? (
       articles
         .filter(result => result.webTitle)
-        .map(
-          ({
-            id,
-            webTitle,
-            webUrl,
-            webPublicationDate,
-            sectionName,
-            fields,
-            pillarId
-          }) => (
-            <FeedItem
-              id={id}
-              key={webUrl}
-              title={webTitle}
-              href={webUrl}
-              publicationDate={webPublicationDate}
-              sectionName={sectionName}
-              pillarId={pillarId}
-              internalPageCode={fields && getId(fields.internalPageCode)}
-              firstPublicationDate={fields.firstPublicationDate}
-              isLive={!fields.isLive || fields.isLive === 'true'}
-              scheduledPublicationDate={fields.scheduledPublicationDate}
-            />
-          )
-        )
+        .map(article => (
+          <FeedItem
+            id={article.id}
+            key={article.webUrl}
+            title={article.webTitle}
+            href={article.webUrl}
+            publicationDate={article.webPublicationDate}
+            sectionName={article.sectionName}
+            pillarId={article.pillarId}
+            internalPageCode={
+              article.fields && getId(article.fields.internalPageCode)
+            }
+            firstPublicationDate={article.fields.firstPublicationDate}
+            isLive={!article.fields.isLive || article.fields.isLive === 'true'}
+            scheduledPublicationDate={article.fields.scheduledPublicationDate}
+            thumbnail={getThumbnail(article, article.frontsMeta.defaults)}
```

Comment:
Is there a way to restructure an object and still have access to it in the fuction? I couldn't figure out the syntax if it exists.

Reply:
not in the function params, I think -- you'd have to pass it in whole and destructure within the function body.